sample_size/scripts/input_utils.py

```diff
@@ -80,9 +78,43 @@ def get_metric_parameters(parameter_definitions: Dict[str, str]) -> Dict[str, fl
     return parameters
 
 
-def get_metric_metadata() -> Tuple[str, Dict[str, float]]:
-    metric_type = get_metric_type().lower()
-    metric_metadata = get_metric_parameters(METRIC_PARAMETERS[metric_type])
-    metric_metadata["mde"] = get_mde(metric_type)
+def get_variants() -> int:
+    number_of_variants = (
+        input("Enter the number of cohorts for this test \n" "Control + number of treatments: ").strip().lower()
+    )
+    if number_of_variants.isdigit():
+        if int(number_of_variants) < 2:
+            raise ValueError("Error: An experiment must contain at least 2 variants.")
+        return int(number_of_variants)
+    elif number_of_variants == "":
+        print("Using default variants(2)...")
+        return 2
+    else:
+        raise ValueError("Error: Please enter an integer for the number of variants.")
+
+
+def register_another_metric() -> bool:
+    register = input("Are you going to register another metric? (y/n)").strip().lower()
+    if register == "y":
+        return True
+    elif register in ["n", ""]:
+        return False
+    else:
+        raise ValueError("Error: Please enter 'y' or 'n'.")
+
+
+def get_metric_metadata() -> Tuple[List[str], List[Dict[str, float]]]:
```

Comment:
We can get one list `metrics` with all data we need to register metrics. 
```suggestion
def get_metrics() -> List[Dict[str, Any]]:
```

Reply:
+1 on this, just make one input parameter with a list of two keys 
```python
[{"metric_type": "", "metric_metadata": <metadata>}]
```